Bon manifestement toutes les PR en cours se font jeter a cause de ça.

Un des tests semble planter ici : https://travis-ci.org/zestedesavoir/zds-site/jobs/50974951

```
FAIL: test_emarkdown (zds.utils.templatetags.tests.test_emarkdown.EMarkdownTest)

----------------------------------------------------------------------

Traceback (most recent call last):

File "/home/travis/build/zestedesavoir/zds-site/zds/utils/templatetags/tests/test_emarkdown.py", line 22, in test_emarkdown

"</blockquote>", tr)

AssertionError: u'<h3>Titre 1</h3>\n<h4>Titre <strong>2</strong></h4>\n<h5>Titre 3</h5>\n<blockquote>\n<p>test</p>\n</blockquote>' != u'<h1>Titre 1</h1>\n<h2>Titre <strong>2</strong></h2>\n<h3>Titre 3</h3>\n<blockquote>\n<p>test</p>\n</blockquote>'
```

Ce qui m'étonne, et bien que c'est moi qui ai ajouté ce test, c'est qu'il n'a aucun sens en réalité. Je comprend pas trop pourquoi on test que le rendu html comporte bien les titres décalés alors que ce n'est pas demandé dans le filtre... J'ai rien changé dans le markdown à ce propos et en réalité ça m'étonne surtout que ça marchait avant ça !

--- a/requirements.txt
+++ b/requirements.txt
@@ -18,7 +18,7 @@ factory-boy==2.4.1
 pygeoip==0.3.2
 pillow==2.7.0
 gitpython==0.3.5
-https://github.com/zestedesavoir/Python-ZMarkdown/archive/2.6.0-zds.2.zip
+https://github.com/zestedesavoir/Python-ZMarkdown/archive/2.6.0-zds.3.zip
 easy-thumbnails==2.2
 
 # Api dependencies

